jaeger-thrift/src/test/java/io/jaegertracing/thrift/internal/senders/UdpSenderTest.java

```diff
@@ -144,7 +144,7 @@ public void testFlushSendsSpan() throws Exception {
     assertEquals(expectedNumSpans, batch.getSpans().size());
 
     io.jaegertracing.thriftjava.Span actualSpan = batch.getSpans().get(0);
-    assertEquals(expectedSpan.context().getTraceId(), actualSpan.getTraceIdLow());
+    assertEquals(expectedSpan.context().getTraceIdLow(), actualSpan.getTraceIdLow());
     assertEquals(0, actualSpan.getTraceIdHigh());
```

Comment:
perhaps this test should be testing High as well, now that we have this upgrade

Reply:
Done